app/polls/welcome/components/PollWelcomePage.js

```diff
@@ -0,0 +1,48 @@
+import { LuUsersRound } from "react-icons/lu";
+import { LuMessagesSquare } from "react-icons/lu";
+import { LuFileChartColumnIncreasing } from "react-icons/lu";
+
```

Comment:
Your build is failing due to the presence of these semicolons. 
Please ensure that you lint your code.